OpenRA.Mods.Common/UpdateRules/Rules/20190314/RemoveAttackTurnDelay.cs

```diff
@@ -0,0 +1,40 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2019 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System.Collections.Generic;
+using System.Linq;
+
+namespace OpenRA.Mods.Common.UpdateRules.Rules
+{
+	public class RemoveAttackTurnDelay : UpdateRule
+	{
+		public override string Name { get { return "Refactor harvester idle behavior."; } }
+		public override string Description
+		{
+			get
+			{
+				return "The AttackTurnDelay parameter has been removed from the AttackAircraft trait in favor of a\n" +
+					   " dynamically calculated attack trajectories.";
+			}
+		}
+
+		readonly List<string> locations = new List<string>();
```

Comment:
This doesn't seem to be used?

Reply:
hmm, Dunno. I just copy pasted this from a different update rule.